src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/oops/AccessFlags.java

```diff
@@ -78,6 +78,6 @@ public void printOn(PrintStream tty) {
 
   // get flags written to .class files
   public int getStandardFlags() {
-    return (int) (flags & JVM_ACC_WRITTEN_FLAGS);
+    return (int)flags;
   }
```

Comment:
This function seems unused.

Reply:
Ah thanks.  Another SA function to remove.